blocks/library/paragraph/index.js

```diff
@@ -389,7 +389,7 @@ export const settings = {
 				},
 			},
 			save( { attributes } ) {
-				return attributes.content;
```

Comment:
We still have an instance of string saving in the parser's unit test but I'm certain it's used at all.